### PR TITLE
V5 ga 9 chip

### PR DESCRIPTION
**Pull request by SrdjanReblaze**
https://reblaze.atlassian.net/jira/software/projects/V5GA/boards/39?selectedIssue=V5GA-9

### CODE REVIEW

File: src/components/CustomTag.vue [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+  &:active {
+    border: 1px solid $color-gray-chateau;
+    color: $color-mako;
+  }
+
+  &:focus {
+    background: $color-white;
+    border: 1px sloid $color-nobel;
+    box-shadow: 0 0 0 2px $color-malibu;
+    color: $color-chalice;
```
Comment (Aviv-Galmidi) [inline src/components/CustomTag.vue:115]: This color isn't set in `colors.scss`

File: src/components/CustomTag.vue [hunk 2]
```diff
@@ -0,0 +1,125 @@
+<template>
+  <div :class="[{disabled: disabled}, variant]">
```
Comment (Aviv-Galmidi) [inline src/components/CustomTag.vue:2]: The component should have a height of 20px

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]
Screen Shot 2023-03-19 at 13 29 31

Where did the outline come from? I cannot find it in the figma

File: src/components/CustomTag.vue (hunk 1 above)
Comment (SrdjanReblaze) [reply src/components/CustomTag.vue]: updated

**Comment by ljubebiza** [comment]
Outline is under fill tag in Figma

Screenshot from 2023-03-28 16-20-25

I also changed sass property for close-svg from display none to visibility hidden in order to prevent tags to move around on hover if there is more than one tag. Please let me know if I should come up with another solution.

**Review by Aviv-Galmidi — CHANGES_REQUESTED** [review]
1. The closing icon covers the text instead of being half-transparent-half-white

```
background: #FFFFFF;
opacity: 0.6;
```

Screen Shot 2023-03-29 at 10 44 31
Screen Shot 2023-03-29 at 10 43 34
Screen Shot 2023-03-29 at 10 43 25


2. It looks like the design is for the closing icon to enlrage the tag a little on hover or click (while displayed) and not always have the space allocated for


3. Found the border, thanks for the clarification

**Review by Aviv-Galmidi — APPROVED** [review]

**Comment by Aviv-Galmidi** [comment]
> Thank you for directing me on right path. I have fixed everything by design. But I am still concerned because tag is not fixed size it will move other elements on page on hover.

I understand where your concern is coming form, this is the design we receieved and would like to go with. We'll keep the fact that the chips would move on hover in mind in our internal testing and get user feedback on the subject

**Pull request merged**